scalableminds/webknossos
# Only allow taskTypeId as parameter in task creation

Basically a revert of #6486

We decided that the small UX improvement of allowing task type id or summary is not worth the harder-to-maintain code after all. The API changes were also not yet used in the wk-libs, which we missed back then. So this revert removes the need to adapt there.

I also fixed the sample organization sample task type in the initial data service, so that during development, bulk task creation can be tested with a fixed csv snippet. (This was one of the original motivations for the change, and is now solved in a much simpler way).

### URL of deployed dev instance (used for testing):
- https://taskcreationnosummary.webknossos.xyz

### Steps to test:
- Create task via bulk creation tab, form creation tab, with NML files and without
- Create task via wk libs
- Should all run through without errors.

------
